hfuzz-signer/Cargo.toml

```diff
@@ -12,14 +12,14 @@ path = "unsigned-message.rs"
 
 [dependencies]
 filecoin-signer = { features = ["with-arbitrary"], path = "../signer" }
-forest_address = { git = "https://github.com/zondax/forest", rev="b1903ba274fc8e5457e71bc88738f17a7a87bc1f" }
-forest_crypto = { git = "https://github.com/zondax/forest", rev="b1903ba274fc8e5457e71bc88738f17a7a87bc1f" }
-forest_message = { git = "https://github.com/zondax/forest", rev="b1903ba274fc8e5457e71bc88738f17a7a87bc1f" }
+forest_address = { git = "https://github.com/zondax/forest", branch="lola/txnid-params-multisig" }
```

Comment:
if this is stable, we should change this to revision hash. To avoid a small change in the branch to have breaking changes on the crate later on.

Created an issue to track this: https://github.com/Zondax/filecoin-rs/issues/185